src/opensearch_sdk_py/settings/settings.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+
+
+from typing import Any, Dict, Optional, Union
+
+from opensearch_sdk_py.transport.stream_input import StreamInput
+from opensearch_sdk_py.transport.stream_output import StreamOutput
+
+
+class Settings:
+    def __init__(
+        self,
+        settings: dict[str, Union[str, Dict]] = {},
+        # TODO: Secure Settings
+    ) -> None:
+        self.settings = settings
+
+    def get(self, setting: str, default: Optional[str] = None) -> str:
+        s_value = self.settings.get(setting)
+        if s_value is None:
+            return default if default else str(None)
+        return str(s_value)
```

Comment:
If you default `default` to `str(None)` I think it avoids an `if`.

Why are we returning `str(None)` anyway? What's that? Why isn't it `None`?